SCloudRandomizer/SearchParams.h

```diff
@@ -7,11 +7,15 @@
 //
 
 #import <Foundation/Foundation.h>
+#import "GlobalData.h"
```

Comment:
I need GlobalData here so I can use one of the helper functions in it to compare entries in the future (ie. null or empty bpm entries)